ReactiveCocoa/Swift/Property.swift

```diff
@@ -15,6 +15,37 @@ public protocol PropertyType {
 	var signal: Signal<Value, NoError> { get }
 }
 
+extension PropertyType {
+	/// Maps each value in the property to a new value.
+	@warn_unused_result
+	public func map<U>(transform: Value -> U) -> AnyProperty<U> {
+		return AnyProperty(
+			initialValue: transform(value),
+			producer: producer.skip(1).map(transform)
+		)
+	}
+
+	/// Combines the latest value of the receiver with the latest value from
+	/// the given property.
+	@warn_unused_result
+	public func combineLatestWith<Other: PropertyType>(otherProperty: Other) -> AnyProperty<(Value, Other.Value)> {
+		return AnyProperty(
+			initialValue: (value, otherProperty.value),
+			producer: producer.combineLatestWith(otherProperty.producer).skip(1)
+		)
+	}
+
+	/// Zips elements of two properties into pairs. The elements of any Nth pair
+	/// are the Nth elements of the two input properties.
+	@warn_unused_result(message="Did you forget to call `start` on the producer?")
+	public func zipWith<Other: PropertyType>(otherProperty: Other) -> AnyProperty<(Value, Other.Value)> {
+		return AnyProperty(
+			initialValue: (value, otherProperty.value),
+			producer: producer.zipWith(otherProperty.producer).skip(1)
```

Comment:
This logic of `skip(1)` specifically is repeated in all the operators. But in general, I think all of these could be defined in terms of a `lift` function that takes a `PropertyType` and a `SignalProducer<T, NoError> -> SignalProducer<R, NoError>` to create the derived property. What do you think?